src/Collapse.jsx

```diff
@@ -13,7 +13,7 @@ function toArray(activeKey) {
   return currentActiveKey;
 }
 
-class Collapse extends Component {
+class Collapse extends PureComponent {
```

Comment:
Don't use PureComponent, we need to keep compatibility in React 15.

Reply:
15.3 is not enough? ant-design require version 16 or higher.

Reply:
ant-design don't require version 16 or higher, we still have test cases for React 15.

Reply:
I saw that:
https://github.com/ant-design/ant-design/blob/master/package.json#L37-L39

```json
  "peerDependencies": {
    "react": ">=16.0.0",
    "react-dom": ">=16.0.0"
  }
```

Reply:
And I also just noticed that there is no peerDependencies in rc-collapse, it's normal?

Reply:
@afc163 I replaced `PureComponent` with `shouldComponentUpdate`.